simpl/exceptions.py

```diff
@@ -107,3 +107,13 @@ def __str__(self):
         """Include custom data in string."""
         return ("Command '%s' returned non-zero exit status %d"
                 % (self.cmd, self.returncode))
+
+
+class SimplConfigException(SimplException):
+
+    """Errors raised by simpl/config."""
+
+
+class SimplConfigUnusedIniOption(SimplConfigException):
+
+    """An option defined in the specified ini file has no match."""
```

Comment:
```
$python -f
Unknown option: -f
```

How about `SimplConfigUnknownOption` and we use it for arguments as well as ini options?